src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -342,7 +349,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             Instant.now());
 
     //send notification
-    sendNotification.sendMsg(parsedMessageContent);
+    Collection tokens = notificationTokenStore.getAllNotificationTokens();
```

Comment:
generics?

Reply:
It's there for now until I combine it with the groups functionality.